source/_components/blink.markdown

```diff
@@ -17,65 +17,100 @@ The `blink` component lets you view camera images and motion events from [Blink]
 
 You will need your Blink login information (username, usually you email address, and password) to use this module.
 
-To set it up, add the following information to your `configuration.yaml` file:
+## {% linkable_title Configuration %}
+
+To enable devices linked in your [Blink](https://blinkforhome.com) account, add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 blink:
   username: YOUR_USERNAME
   password: YOUR_PASSWORD
+  scan_interval: 60
+  binary_sensors:
```

Comment:
We usually start with a minimum required config example. A more complex example can follow the config variables spec.